pkg/controllers/uiplugin/controller.go

```diff
@@ -158,6 +158,13 @@ func (rm resourceManager) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		return ctrl.Result{}, err
 	}
 
+	if pluginInfo.Korrel8rImage != "" {
+		var OpenshiftLoggingNs = "openshift-logging"
+		var OpenshiftNetobservNs = "netobserv"
+		pluginInfo.LokiServiceNames[OpenshiftLoggingNs], _ = getLokiServiceName(ctx, rm.k8sClient, OpenshiftLoggingNs)
```

Comment:
Not to block this PR, but maybe we can pass the context and client to the `PluginInfoBuilder` so it can discover the Loki services there